src/Form/AdTechMediaConfigForm.php

```diff
@@ -0,0 +1,304 @@
+      '#title' => $this->t('Email Address'),
+      '#default_value' => $config->get('email'),
+      '#title_display' => 'after',
+    );
+
+    $form['general']['activate'] = array(
+      '#type' => 'button',
+      '#value' => $this->t('Activate'),
+      '#attributes' => array(
+        'class' => array('btn', 'activate'),
+      ),
+    );
+
+    $form['content'] = array(
+      '#type' => 'details',
+      '#title' => $this->t('Content configuration'),
+      '#open' => TRUE,
+    );
+
+    $form['content']['content_pricing'] = array(
+      '#type' => 'textfield',
+      '#title' => $this->t('Content pricing'),
+      '#default_value' => $config->get('content_pricing'),
+      '#description' => $this->t('It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout. The point of using Lorem Ipsum is that it has a more-or-less normal distribution of letters'),
+      '#placeholder' => $this->t('number'),
+      '#title_display' => 'after',
+    );
+
+    $form['content']['content_currency'] = array(
+      '#type' => 'select',
+      '#title' => $this->t('Content currency'),
+      '#options' => array(
+        'usd' => $this->t('USD'),
+        'mdl' => $this->t('MDL'),
+      ),
+      '#default_value' => $config->get('content_currency'),
+      '#description' => $this->t('Select content currency.'),
+    );
+
+    $form['content']['content_paywall'] = array(
+      '#type' => 'textfield',
+      '#title' => $this->t('Content paywall'),
+      '#default_value' => $config->get('content_paywall'),
+      '#description' => $this->t('Remove paywall after.'),
+      '#placeholder' => $this->t('number'),
+      '#title_display' => 'after',
+    );
+
+    $form['content']['content_paywall_type'] = array(
+      '#type' => 'select',
+      '#title' => $this->t('Content paywall type'),
+      '#options' => array(
+        'transactions' => $this->t('Transactions'),
+      ),
+      '#default_value' => $config->get('content_paywall_type'),
+      '#description' => $this->t('Select preview type.'),
+    );
+
+    $form['content']['content_preview'] = array(
+      '#type' => 'textfield',
+      '#title' => $this->t('Content preview'),
+      '#default_value' => $config->get('content_preview'),
+      '#description' => $this->t('Select how content should be displayed.'),
+      '#placeholder' => $this->t('number'),
+      '#title_display' => 'after',
+    );
+
+    $form['content']['content_preview_type'] = array(
+      '#type' => 'select',
+      '#title' => $this->t('Content preview type'),
+      '#options' => array(
+        'paragraph' => $this->t('Paragraph'),
+      ),
+      '#default_value' => $config->get('content_preview_type'),
+      '#description' => $this->t('Select preview type.'),
+    );
+
+    $form['content']['locking_algorithm'] = array(
+      '#type' => 'select',
+      '#title' => $this->t('Content unlocking algorithm'),
+      '#options' => array(
```

Comment:
Please refer to the available options bellow:
blur, scramble, keywords, blur+scramble